docs/source/reference/series.rst

```diff
@@ -204,3 +204,43 @@ Serialization / IO / Conversion
    Series.to_csv
    Series.to_excel
 
+Plotting
+-------------------------------
+``Series.plot`` is both a callable method and a namespace attribute for
+specific plotting methods of the form ``Series.plot.<kind>``.
+
+.. autosummary::
+   :toctree: api/
+
+   Series.plot.bar
+   Series.plot.box
+   Series.plot.hist
+
+.. autosummary::
+   :toctree: api/
+
+   Series.hist
+
+Datetime Methods
+----------------
```

Comment:
why are datetime methods included?

Reply:
My mistake... I've updated it. But, this time, as I inserted the plot functions after the other accessors, I ended up moving the conversion methods and they appear as both deleted and included on the PR.